tests/test_bundle.py

```diff
@@ -726,6 +726,7 @@ def put_bundle(
                 }
             )
             self.assertIn('version', resp_obj.json)
+            self.assertIn('manifest', resp_obj.json)
```

Comment:
add verification for the following manifest fields `name`,`uuid`,`version`,`indexed`